app/interactors/create_group.rb

```diff
@@ -1,2 +1,13 @@
 class CreateGroup < StandardInteraction
+  def validate_input
+    context.fail!(errors: "invalid input") unless context.group_params
+  end
+
+  def execute
+    context.group = context.user.groups.create(context.group_params)
+  end
```

Comment:
@Enriikke I think this is the cleanest way to create a `Group` and a `UserGroupMembership` at the same time. `context.user` is my `current_user`, which I think is alright and I can't think of an instance where that will be a problem right now but perhaps you have thoughts?